net/ddns-scripts/Makefile

```diff
@@ -57,6 +57,17 @@ define Package/ddns-scripts-services/description
   Dynamic DNS Client definitions for supported services
 endef
 
+define Package/ddns-scripts-utils
+  $(call Package/ddns-scripts/Default)
+  TITLE:=Utility scripts for ddns configs
+  DEPENDS:=ddns-scripts
+  PROVIDES:=ddns-scripts_utils
```

Comment:
I would say this is not necessary. Please remove `PROVIDES:=ddns-scripts_utils`, this is only needed if there are two packages that provides the same functionality, but build with different feature sets.

Reply:
I see! Just out of curiosity, why are the other packages in this makefile using `PROVIDES`?

Reply:
As far as I can remember, this is because the names were changed and in order not to break downstream projects, PROVIDES was added with the old name